src/BurnableMintableCappedERC20.sol

```diff
@@ -18,7 +27,15 @@ contract BurnableMintableCappedERC20 is MintableCappedERC20 {
         string memory symbol,
         uint8 decimals,
         uint256 capacity
-    ) MintableCappedERC20(name, symbol, decimals, capacity) {}
+    ) MintableCappedERC20(name, symbol, decimals, capacity) {
+        DOMAIN_SEPARATOR = keccak256(
+            abi.encode(
+                keccak256('EIP712Domain(uint256 chainId,address verifyingContract)'),
```

Comment:
Why is this not a constant?

Reply:
Good idea! I will move that hash in constants